WD-Firmware/board/nexys_a7_swerv1/bsp/printf.c

```diff
@@ -0,0 +1,396 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2019 Western Digital Corporation or its affiliates.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+// Based on a public domain version of printf
+// Web link: http://www.rte.se/sites/default/files/Blog/Modesty/ee_printf.c 
+//
+
+
```

Comment:
general comment - the file implementation doesn't follow the coding convention

Reply:
yes, not in this phase